hyperspy/tests/model/test_components2D.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import nose.tools as nt
+
+import hyperspy.api as hs
+
+
+class TestGaussian2D:
```

Comment:
This class should be renamed and/or moved to a different file. At the moment it claims to test `Gaussian2D`, however tests `Model2D` fitting using identical gaussian, which can in principle be a big number of functions and not necessarily a gaussian.

The test itself is useful, to test fitting, however an additional test is required to compare the gaussian function output with some defined array to check that it actually works